src/collectd.conf.pod

```diff
@@ -1415,6 +1415,92 @@ number.
 
 =back
 
+=head2 cURL Statistics
+
+All cURL-based plugins support collection of generic, request-based
+statistics. These are disabled by default and can be enabled selectively for
+each page or URL queried from the curl, curl_json, or curl_xml plugins. See
+the documentation of those plugins for specific information. This section
+describes the available metrics that can be configured with each plugin. All
+options are disabled by default.
+
+See L<http://curl.haxx.se/libcurl/c/curl_easy_getinfo.html> for more details.
+
+=over 4
+
+=item B<total_time> B<true|false>
```

Comment:
If you want to be able to point people to the library docs to read details about the timing informations, using a naming similar to curl makes sense imho.
CURLINFO_TOTAL_TIME being total_time is pretty obvious.